core/controllers/base.py

```diff
@@ -224,7 +234,12 @@ def __init__(self, request, response):  # pylint: disable=super-init-not-called
             self.payload = None
 
     def dispatch(self):
-        """Overrides dispatch method in webapp2 superclass."""
+        """Overrides dispatch method in webapp2 superclass.
+
+            Raises:
+                CSRF token Exception: If CSRF token is missing
```

Comment:
Exception: The CSRF token is missing.
UnauthorizedUserException: The CSRF token is invalid.